test/__snapshots__/Navbar.spec.js.snap

```diff
@@ -263,7 +263,7 @@ exports[`<Navbar /> can have custom sidenav 1`] = `
       className="nav-wrapper"
     >
       <a
-        className="brand-logo"
+        className="brand-logo right"
```

Comment:
How come this snapshot was updated if `alignLinks` is not left?

Reply:
[The test, "can have custom sidenav" is left aligned](https://github.com/react-materialize/react-materialize/blob/5a5803e715b32d5f8193425fef36d3fdae5e3917/test/Navbar.spec.js#L121)